src/wallet/wallet.h

```diff
@@ -745,8 +749,13 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      * selected by SelectCoins(); Also create the change output, when needed
      * @note passing nChangePosInOut as -1 will result in setting a random position
      */
+<<<<<<< 08b37c5e06bf1698c1d0e5905806382cbfa8cefd
```

Comment:
@jonasschnelli This commit is still in the history.

You didn't solve the conflict in this commit but in a later one. Looks weird if someone digs through it.